src/chunk_append/exec.h

```diff
@@ -9,7 +9,14 @@
 #include <postgres.h>
 #include <nodes/bitmapset.h>
 #include <nodes/extensible.h>
+
+#include "compat.h"
+
+#if PG12_LT /* nodes/relation.h renamed in fa2cf16 */
```

Comment:
Does the hash-ID here refer to a commit? Is it a commit in PG or in our code? Please clarify. 

Also, maybe better with a link in that case.